src/main/webapp/availabilityTable.jsp

```diff
@@ -0,0 +1,24 @@
+<%@ page import="com.google.sps.data.AvailabilityTimeSlots" %>
+<%
+  AvailabilityTimeSlots list = new AvailabilityTimeSlots(request.getParameter("timeZoneOffset"));
+  pageContext.setAttribute("list", list);
+%>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %> 
+
+<table class="table table-sm text-center">
+  <thead>
+    <tr>
+      <th scope="col">${list.getTimeSlots().get(0).getDate()}</th>
+    </tr>
+  </thead>
+  <tbody>
+    <!-- TODO: Allow clicking and scrolling over multiple slots to select them.-->
+    <c:forEach items = "${pageScope.list.getTimeSlots()}" var = "timeSlot">
+      <tr>
+        <td onclick="switchTile(this)" data-utc="${timeSlot.getUTCEncoding()}" class="${timeSlot.getClassName()}">
```

Comment:
Replace getClassName with a ternary (per my other comment, to keep view logic in the jsp)?

Reply:
Done